benchmark/es/foreach-bench.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const common = require('../common.js');
+
+const bench = common.createBenchmark(main, {
+  method: ['array', 'forEach'],
```

Comment:
I think just 'for' would be more descriptive since they are both iterating over an array.